sql/src/test/java/io/crate/integrationtests/SysCheckerIntegrationTest.java

```diff
@@ -61,6 +57,7 @@ public void testMinimumMasterNodesCheckWithResetCorrectSetting() {
         int setMinimumMasterNodes = numberOfMasterNodes() / 2 + 1;
         execute("set global discovery.zen.minimum_master_nodes=?", new Object[]{setMinimumMasterNodes});
         SQLResponse response = execute("select severity, passed from sys.checks where id=?", new Object[]{1});
+        assertThat(response.rowCount(), equalTo(1L));
```

Comment:
I added this assertion because when changed the SysChecker IDs this test was failing with an array index out of bounds exception, which was not much meaningful